defaults/test_default.yaml

```diff
@@ -8,7 +8,8 @@ scylla_mgmt_repo: 'http://repositories.scylladb.com/scylla/repo/qa-test/centos/s
 
 experimental: 'true'
 
-append_scylla_args: '--blocked-reactor-notify-ms 500 --abort-on-lsa-bad-alloc 1 --abort-on-seastar-bad-alloc'
```

Comment:
What do you change the defaults?
If you want to override it, you need to specify it in your test yaml

Reply:
No. This change s temporary and will be removed. The PR is not ready for review